packages/ui/src/components/forms/file-upload/components/FileInfo.vue

```diff
@@ -269,6 +286,17 @@ export const FileInfo = /*#__PURE__*/ Vue.extend({
     },
     cancel() {
       this.controller.abort();
+    },
+    checkCustomFileName(value: string) {
```

Comment:
kinda wondering if we should turn this into an event instead, to allow potentially allow people to plug in their own validation into this

Reply:
Ye could point can't this use the same validation logic and other form fields where you pass it from vee validate?

Reply:
I have changed this to pass in as prop, I think there was something around the forge inline editor that was a bit strange. @JakeMinor was planning to have a look at it and make it better so maybe we could change it when he gets around to that.